openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -2297,7 +2299,7 @@ def test_rate_limiting_registration_view(self):
         to enforce limits; that's why this test needs a "real"
         default cache (as opposed to the usual-for-tests DummyCache)
         """
-        for _ in range(RegistrationValidationThrottle().num_requests):
+        for _ in range(int(settings.REGISTRATION_RATELIMIT.split('/')[0])):
             self.request_without_auth('post', self.path)
```

Comment:
Add an assert inside for loop after the request that the response code in not 403 until the rate limit is no exceeded.